components/ws-manager-mk2/main.go

```diff
@@ -87,7 +87,7 @@ func main() {
 		Port:                   9443,
 		HealthProbeBindAddress: cfg.Health.Addr,
 		LeaderElection:         enableLeaderElection,
-		LeaderElectionID:       "ws-manager-mk2.gitpod.io",
+		LeaderElectionID:       "ws-manager-mk2-leader.gitpod.io",
```

Comment:
this ID shows up as a ConfigMap, including `leader` makes it more obvious that the configmap is used for leader-election